bench-tx/bench-tx.json

```diff
@@ -1,21 +1,21 @@
 {
   "simple": {
-    "prologue": 3644,
-    "notes_processing": 1151,
+    "prologue": 3833,
+    "notes_processing": 2178,
     "note_execution": {
-      "0x47b2fbff8a3f09e40343238e7b15c8918d7c63e570fd1b3c904ada458c4d74bd": 392,
-      "0x8a55c3531cdd5725aa805475093ed3006c6773b71a008e8ca840da8364a67cd6": 715
+      "0x8cb51db3dbec8fab6c0ce3ac3e332aaa28fc99de07ae5a52dd93047a1fe459c2": 1401,
```

Comment:
I think an order of the notes has changed, so both of this notes now require twice as much cycles as it was before. I think so because both notes are essentially the same, with the only difference that one note generates one output note, and the other generates two of them. With this in mind, the change in cycles number becomes consistent:
`392 -> 735`
`715 -> 1401`